.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm' # See documentation for possible values
+    directory: '/' # Location of package manifests
```

Comment:
Does this also check the sub dirs?

Reply:
Not sure if it's desirable to have that level of alertness for a mere example-app. What are your thoughts on this?

Reply:
True! And vulnerabilities are tracked either way.